frontend/packages/dev-console/integration-tests/features/Knative/connector-Event-Sources-ODC-4230.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Connector Event Sources
+    User should be able to install Connector Event Sources
+
+
+#Scenarios of installing these operator has been included in operators.feature file
+Background: 
+    Given user has installed the OpenShift Serverless Operator
+    And user has created knative serving
+    And user has create knative eventing
+    And user has installed Knative Apache Camel Operator
```

Comment:
IMO name of operator is just `knative camel Operator`

Reply:
No. It's Knative Apache Camel Operator.